core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.component.html

```diff
@@ -83,15 +83,24 @@
         </a>
       </td>
       <td>
-        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary text-center">
+        <a (click)="topicRights.canEditTopic() && openStoryEditor(summary.getId())" class="story-list-summary chapter-count text-center">
           <span> {{ summary.getPublishedChaptersCount() }} Published</span>
-          <span> {{ summary.getTotalChaptersCount() }} Total</span>
+          <span *ngIf="summary.getTotalChaptersCount()!=summary.getPublishedChaptersCount()"> {{ summary.getTotalChaptersCount() - summary.getPublishedChaptersCount() }} Awaiting Publication</span>
```

Comment:
Do not put complicated if conditions in HTML -- at most, call a single function like areAnyChaptersUnpublished().  Ditto elsewhere below.

(See other files in the existing codebase.)

Reply:
Done